det-keygen.md

```diff
@@ -194,3 +194,244 @@ Equivalent processes are explicitly allowed by point 4 of [FIPS 186-5, Appendix
 [RFC 6979]: https://rfc-editor.org/rfc/rfc6979.html
 [RFC 6979, Section 2.3.2]: https://rfc-editor.org/rfc/rfc6979.html#section-2.3.2
 [draft-irtf-cfrg-det-sigs-with-noise-04]: https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-det-sigs-with-noise-04
+
+## RSA
+
+At a high level, this RSA key generation process involves instantiating
+HMAC_DRBG with SHA-256, the seed, and a personalization string including the
+target bit size; converting byte strings from the DRBG into prime number
+candidates until two primes are generated; and restarting if the resulting
+totient is not coprime with the fixed public exponent.
+
+Below, HMAC(K, M) denotes the 32-byte HMAC of message M with key K computed with
+hash function SHA-256.
+
+**Input**:
+
+1. *seed* — A byte string. The seed MUST be at least 128 bits (16 bytes) long,
+   and SHOULD include at least 192 bits of entropy to provide a security margin
+   against multi-target attacks.
+
+2. *bits* — The desired modulus size, in bits. *bits* MUST be a multiple of 16,
+   and MUST be at least 2048.[^sizes]
+
+**Output**:
+
+1. *N*, *e*, *d*, *p*, *q* — The generated modulus, public exponent,
+   private exponent, and prime factors.
+
+**Process**:
+
+1. *personalization_string* = `det RSA key gen` || I2OSP(*bits*, 2)
```

Comment:
NitPick: I like having a distinguished terminal character when I'm doing domain separation. That way I don't need to worry about accidentally creating two strings where one is a prefix of another.
```suggestion
1. *personalization_string* = `det RSA key gen:` || I2OSP(*bits*, 2)
```